docs/application/dotnet/tutorials/details/release.md

```diff
@@ -0,0 +1,6 @@
+<a id="release"></a>
```

Comment:
Do we need this <a id="release"></a>?
In markdown files, we do not use these tags.
Please verify and change.

Reply:
I remove this page as journey2w's request.